### ISSUE TITLE
Open Api Document

### ISSUE THREAD

**Issue by ArieGato** [soundcloud/api#217]
It's really great that there is a Swagger UI, but is there an endpoint to the Open Api document? I would like to generate a Client with NSwag from the json or yaml document.

**Comment by davidjohnson85** [comment]
Yes, you can access the OpenAPI (formerly known as Swagger) document for an API by appending /swagger.json or /swagger.yaml to the base URL of the API. This will give you the JSON or YAML representation of the API specification, respectively.

For example, if your API is hosted at https://api.example.com, you can access the OpenAPI document by making a request to https://api.example.com/swagger.json or https://api.example.com/swagger.yaml.

Once you have obtained the JSON or YAML document, you can use tools like NSwag to generate a client library or perform other code generation tasks based on the API specification.

Note that the availability of the OpenAPI document depends on the API provider, and they may choose to expose it or not. If you are unable to find the OpenAPI document for a specific API, you can reach out to the API provider or check their documentation for further guidance.

**Regards;
_David Johnson._**

**Comment by dpreussler** [comment]
Here you go! Added a link to https://developers.soundcloud.com/docs/api/explorer/
